.github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Standard Issue Template for POs
+about: Standard Issue Template to be used by PUL POs
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Summary (Product Owner)
+
+### User Story (Product Owner, when applicable)
+_As a [persona], I want to [describe intent], so that [explain the big problem you are trying to solve and its benefit]_
+
+### Impact (Product Owner)
+_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
+
+### Priority recommendation (Product Owner)
+
+- [ ] asap
+- [ ] within the next 3 weeks
+- [ ] PO will prioritize
+
+## Sudden Priority Justification (Product Owner)
+_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "Maintenance/Research" labels_
```

Comment:
We have moved away from the maintenance/research board.

```suggestion
_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "dls-work-cycle" labels_
```

Reply:
sounds good!